### PR TITLE
chore: adds vanity import check.

### PR DESCRIPTION
**Pull request by jcchavezs**
This PR adds check on CI to verify vanity imports are rightfully added.

### CODE REVIEW

**Comment by pellared** [comment]
Have you tried to add https://github.com/jcchavezs/porto/ to https://github.com/golangci/golangci-lint?

File: Makefile [hunk 1]
```diff
@@ -136,6 +139,10 @@ lint: misspell lint-modules | $(GOLANGCI_LINT)
 	    $(GOLANGCI_LINT) run); \
 	done
 
+.PHONY: vanity-import-check
+vanity-import-check: | $(MISSPELL)
```
Comment (MrAlias) [inline Makefile:143]:
```suggestion
vanity-import-check: | $(PORTO)
```

**Review by jmacd — APPROVED** [review]
:100:

**Comment by jcchavezs** [comment]
@pellared that was actually an idea but first I wanted to see this in action here.

**Comment by jcchavezs** [comment]
This is failing now on purpose. I wanted it to fail to test the setup.

**Comment by jcchavezs** [comment]
Do we need to update changelog?

**Review by Aneurysm9 — APPROVED** [review]

**Pull request merged**